src/EditorFeatures/CSharp/Formatting/Indentation/SmartTokenFormatter.cs

```diff
@@ -50,9 +50,10 @@ internal class SmartTokenFormatter : ISmartTokenFormatter
             var common = startToken.GetCommonRoot(endToken);
 
             // if there are errors, do not touch lines
-            // Exception: In the case of try-catch-finally block, a try block without a catch/finally block is considered incomplete
+            // Exception 1: In the case of try-catch-finally block, a try block without a catch/finally block is considered incomplete
```

Comment:
The method name is self-explanatory so having the comment at the callsite seems more reasonable.

If the list becomes longer and the method name becomes a generic then we can move the exception list next to the exceptions